app/src/main/res/layout/fragment_order.xml

```diff
@@ -151,16 +151,15 @@
                     android:text="@string/invite"
                     android:textColor="@color/black" />
 
-                <com.google.android.material.floatingactionbutton.FloatingActionButton
+                <ImageButton
                     android:id="@+id/share"
                     android:layout_width="@dimen/fab_width"
                     android:layout_height="@dimen/fab_height"
                     android:layout_margin="@dimen/layout_margin_medium"
-                    android:background="@android:color/white"
+                    android:background="@drawable/ic_share_grey_24dp"
                     android:scaleType="center"
                     app:backgroundTint="@android:color/white"
                     app:srcCompat="@drawable/ic_share_grey_24dp" />
```

Comment:
Unneeded

Reply:
Totally agree with whatever you said. But I have no idea why are you saying it as a response to my code review? Care to explain?

Reply:
Okay, I get it I didn't notice srcCompat already had the image. I'm really sorry, will remove the background line.